### PR TITLE
package: lock all dependencies

### PR DESCRIPTION
**Pull request by danstarns**
Now have merged dependabot: 

* https://github.com/GrapesJS/grapesjs/pull/6069

Let's lock the dependencies for more consistency and rely on auto-updates along with a ci run to assert updated packages. 

Related: 
* https://github.com/GrapesJS/grapesjs/issues/6049

### CODE REVIEW

**Review by artf — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -1551,10 +1551,10 @@
   dependencies:
     "@babel/types" "^7.20.7"
 
-"@types/backbone@^1.4.15":
-  version "1.4.19"
-  resolved "https://registry.npmjs.org/@types/backbone/-/backbone-1.4.19.tgz#f6e8406fed40ca3fe224f6e59115142b62be76d6"
-  integrity sha512-byyn236JymGByOajKA7mi1k+/jKn162TIvArOB4SHgOGbVlFj8CSfJH4jekP0qo0vJwW5khrrsiiO1Jsos6ZvA==
+"@types/backbone@1.4.15":
```
Comment (delagen) [inline yarn.lock:1554]: @artf Seems need to fix to 1.4.19, because 1.4.15 cause error with underscore typings

Comment (danstarns) [reply yarn.lock:1554]: hi @Delagen can you provide a replication of your issue?

Comment (delagen) [reply yarn.lock:1554]: I have this error
```
TS2694: Namespace '".../node_modules/@types/und
erscore/index"' has no exported member 'ListIterator'.
```

With 1.4.19 I have no error. With 1.4.20 I have another error caused View.events different declaration

Comment (delagen) [reply yarn.lock:1554]: May be It caused by "module": "ESNext","useDefineForClassFields": true,

**Pull request merged**